inspire/base/templates/format/record/Conference_HTML_detailed.tpl

```diff
@@ -0,0 +1,104 @@
+{#
+## This file is part of INSPIRE.
+## Copyright (C) 2015 CERN.
+##
+## INSPIRE is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## INSPIRE is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{% extends "format/record/Default_HTML_detailed.tpl" %}
+
+{% block header %}
+  <div class="row">
+    <div class="col-md-12">
+      <h3>
+      {% for collection in record['collections'] %}
+        {% if 'primary' in collection %}
+          <span class="label label-default pull-left {% if not loop.first %} collection-primary {% endif %}">
+          {{ collection['primary'] }}</span>
+        {% endif %}
+      {% endfor %}
+      </h3>
+    </div>
+  </div>
+  <div class="row"><div class="col-md-12"><p></p></div></div>
+  <div class="row"><div class="col-md-12"><p></p></div></div>
+  <div class="row">
+    <div class="col-md-12">
+     <div class="record-title pull-left">
+      {% if record['title']|is_list %}
+        {% for title in record['title'] %}          
+          {{ title|capitalize }}
+        {% endfor %}
+      {% else %}
+        {{ record['title']|capitalize }}
+      {% endif %}
+      {% if record['acronym'] %}
+        ({{ record['acronym'] }}) 
+      {% endif %}
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      <div class="pull-left">
+      {{ record|conference_date }}
+      {{ record['place'] }}
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      <div class="pull-left">
+        <label >CNUM:</label> {{ record['cnum'] }}
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-12">
+      <a class="pull-left" href="#">Contributions</a>
+    </div>
+  </div>
+  {% if record['url'] %}
+    <div class="row">
+      <div class="col-md-12">
+        <div class="pull-left">
+          <label>URL:</label><a href="{{ record['url'][0] }}"> {{ record['url'][0] }}</a>
+        </div>
+      </div>
+    </div>
+  {% endif %}
+  {% if record['contact_email'] %}
+    <div class="row">
+      <div class="col-md-12">
+        <div class="pull-left">
+          <!-- FIXME email in record.py -->
```

Comment:
Right now nothing because we merged the brief results, but I didn't had time today to update this PR. I would take care tomorrow and I will push again.